### DISCUSSION TITLE
应用的单点登录实现了嘛？没看到对应文档

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#213] [02.提问]
<img width="1040" alt="image" src="https://user-images.githubusercontent.com/6092525/199431168-b0f2da78-ecd4-4924-bc08-1d9b6f38e2a1.png">
这块描述的单点登录实现了嘛，如果没实现的话 用什么技术方案能说下嘛

**Comment by zuihou** [comment]
没有
